notebooks/223-text-prediction/223-text-prediction.ipynb

```diff
@@ -766,14 +766,15 @@
     "            return history\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #47.            demo.launch()
</pre><p>It would be grate to add&nbsp;&nbsp;&nbsp;&nbsp;</p><p> </p><p>	try:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;demo.launch()</p><p>&nbsp;&nbsp;&nbsp;&nbsp;except Exception:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;demo.launch(share=True)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
Done